src/Voron/Impl/Paging/RvnMemoryMapPager.cs

```diff
@@ -89,7 +89,7 @@ public RvnMemoryMapPager(StorageEnvironmentOptions options, VoronPathSetting fil
         {
             var state = pagerState ?? _pagerState;
 
-            if (SysInfo.CanPrefetch)
+            if (Options.EnablePrefetching && SysInfo.CanPrefetch)
```

Comment:
I think only one is needed?

Reply:
@ppekrol you should not prefetch in non prefetchable environment even if EnablePrefetching is enabled.  So I think the above change is fine.

Reply:
ok.. @ppekrol - you are right.
@haludi -  `override protected virtual bool CanPrefetchQuery()`
Implement it using both `SysInfo.CanPrefetch` and `_canPrefetchAhead` (make sure it _can_ be overriden by c'tor, i.e. - even if we are on Win10 and we are able to prefetch, but 32 bit pager wants this feature off)